### PR TITLE
Update React-spring

### PR DESCRIPTION
**Pull request by kierangillen**
- Updates `React-spring` to `8.0.1`
- Adds `.docz/*` to gitignore

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-palette* ready!

Built with commit 9748ca22ba13ca37a8a0fa938ef5960d7c59e490

https://deploy-preview-298--artsy-palette.netlify.com

File: www/src/components/Sidebar/NavTree.tsx [hunk 1]
```diff
@@ -210,9 +210,7 @@ const NavLink = styled(NavLinkWrapper)<{ expandSubNav?: boolean }>`
     return `
       cursor: ${cursor};
     `
-  }}
-
-  &:hover {
+  }} &:hover {
```
Comment (damassi) [inline www/src/components/Sidebar/NavTree.tsx]: A `;`  after `}}` will fix this collapse

**Comment by l2succes** [comment]
Looks good but like as mentioned offline, let's test this using `yarn watch:emission` locally first to see if it breaks anything.

**Comment by zephraph** [comment]
.docz shouldn't exist anymore, right?

**Comment by damassi** [comment]
@kierangillen If this has stalled would you mind closing / completing? (Might be easier to start from scratch since we've restructured palette since this was opened.)

**Comment by l2succes** [comment]
Yeah let's close it. We just got through the react-native upgrade in emission so we're back at upgrading palette as well anyway

**Pull request closed without merging**